### PR TITLE
Chinese Simplified Created

### PR DESCRIPTION
**Pull request by boblone19**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [ ] Your PR is linked to an issue.
- [ ] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [ ] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [ ] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR covers issue #< insert number here >.

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

File: Document-zhcn/0x02-Frontispiece.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+<br><br>
+
+| 语言 |翻译和审稿 |
+| --- | --- |
+| 中文 | Peter Chi, and Lex Chien, Henry Hu, Leo Wang |
+| 法语 | Romuald Szkudlarek, Christian Dong (Review) |
+| 德文 | Rocco Gränitz, Sven Schleier (Review) |
+| 西班牙语 | Martin Marsicano, Carlos Holguera |
+| 日语 | Koki Takeyama, Riotaro Okada (Review) |
+| 俄语 | Gall Maxim, Chelnokov Vladislav (Review), Oprya Egor (Review), Tereshin Dmitry (Review) |
```
Comment (sushi2k) [inline Document-zhcn/0x02-Frontispiece.md]: Please add the Korean team korean translation (was just added a few weeks back)

File: Document-zhcn/0x02-Frontispiece.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
```
Comment (sushi2k) [inline Document-zhcn/0x02-Frontispiece.md]: Just replace foo with the chinese tranlsation of Korean :-)
```suggestion
| foo | Youngjae Jeon, Jeongwon Cho, Jiyou Han, Jiyeon Sung |
```

File: README-zhcn.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
+<a href="https://github.com/OWASP/owasp-masvs/releases/download/1.1/OWASP_Mobile_AppSec_Verification_Standard_v1.1.pdf"><img width="25%" align="right" style="float: right;" src="Document/images/masvs-mini-cover.png"></a>
```
Comment (sushi2k) [inline README-zhcn.md]: Can you please move this file to the Document-zhcn folder?

File: README-zhcn.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+- [V1: 架构，设计和威胁建模要求](Document-zhcn/0x06-V1-Architecture_design_and_threat_modelling_requireme.md)
+- [V2: 数据存储和隐私要求](Document-zhcn/0x07-V2-Data_Storage_and_Privacy_requirements.md)
+- [V3: 密码学要求](Document-zhcn/0x08-V3-Cryptography_Verification_Requirements.md)
+- [V4: 身份验证和会话管理要求](Document-zhcn/0x09-V4-Authentication_and_Session_Management_Requirements.md)
+- [V5: 网络通信要求](Document-zhcn/0x10-V5-Network_communication_requirements.md)
+- [V6: 环境互动要求](Document-zhcn/0x11-V6-Interaction_with_the_environment.md)
+- [V7: 代码质量和构建设置要求](Document-zhcn/0x12-V7-Code_quality_and_build_setting_requirements.md)
+- [V8: 针对逆向工程要求的弹性](Document-zhcn/0x15-V8-Resiliency_Against_Reverse_Engineering_Requirements.md)
+- [附录A：词汇表](Document-zhcn/0x90-Appendix-A_Glossary.md)
+- [附录B：参考](Document-zhcn/0x91-Appendix-B_References.md)
```
Comment (sushi2k) [inline README-zhcn.md]: Once moved, please remove the "Document-zhcn" from the links

File: Document-zhcn/0x02-Frontispiece.md (hunk 1 above)
Comment (boblone19) [reply Document-zhcn/0x02-Frontispiece.md]: Will do thx for reminders

File: README-zhcn.md (hunk 3 above)
Comment (boblone19) [reply README-zhcn.md]: removed

File: README-zhcn.md (hunk 4 above)
Comment (boblone19) [reply README-zhcn.md]: mission completed

File: Document-zhcn/0x02-Frontispiece.md (hunk 2 above)
Comment (boblone19) [reply Document-zhcn/0x02-Frontispiece.md]: did it

**Comment by boblone19** [comment]
forget about it

File: Document-zhcn/0x02-Frontispiece.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+![OWASP LOGO](images/OWASP_logo.png)
+
+# 移动应用安全验证标准
+
+## 关于标准
```
Comment (SpZanG) [inline Document-zhcn/0x02-Frontispiece.md]:
```suggestion
## 关于此标准
```

File: Document-zhcn/0x02-Frontispiece.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+欢迎使用移动应用安全验证标准（MASVS）1.1. MASVS 是社区项目，宗旨是建立一个安全需求框架，用于在iOS和Android上的设计、开发和测试安全的移动应用程序。
```
Comment (SpZanG) [inline Document-zhcn/0x02-Frontispiece.md]:
```suggestion
欢迎阅读移动应用安全验证标准（MASVS）1.1版. MASVS 是一个社区项目，其宗旨是建立一个安全需求的框架给人们去设计、开发和测试iOS和Android的移动应用程序。

File: Document-zhcn/0x02-Frontispiece.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+MASVS是社区努力和行业反馈的结晶。我们希望该标准会随着时间的推移而发展，并欢迎社区提供反馈。
```
Comment (SpZanG) [inline Document-zhcn/0x02-Frontispiece.md]:
```suggestion
MASVS是社区努力和行业反馈的成果。我们希望该标准会随着时间的推移不断更新，并且非常欢迎来自社区的意见反馈。
```

File: Document-zhcn/0x02-Frontispiece.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+与我们联系的最佳方法是通过 OWASP Mobile Project Slack频道: <https://owasp.slack.com/messages/project-mobile_omtg/details/> .
```
Comment (SpZanG) [inline Document-zhcn/0x02-Frontispiece.md]:
```suggestion
与我们取得联系的最佳方式是通过 OWASP Mobile Project Slack频道: <https://owasp.slack.com/messages/project-mobile_omtg/details/>
```

File: Document-zhcn/0x02-Frontispiece.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+
+可以在以下URL上创建帐户: [https://owasp.slack.com/](https://join.slack.com/t/owasp/shared_invite/enQtNjExMTc3MTg0MzU4LWQ2Nzg3NGJiZGQ2MjRmNzkzN2Q4YzU1MWYyZTdjYjA2ZTA5M2RkNzE2ZjdkNzI5ZThhOWY5MjljYWZmYmY4ZjM).
```
Comment (SpZanG) [inline Document-zhcn/0x02-Frontispiece.md]:
```suggestion
可以通过以下URL创建帐户: [https://owasp.slack.com/](https://join.slack.com/t/owasp/shared_invite/enQtNjExMTc3MTg0MzU4LWQ2Nzg3NGJiZGQ2MjRmNzkzN2Q4YzU1MWYyZTdjYjA2ZTA5M2RkNzE2ZjdkNzI5ZThhOWY5MjljYWZmYmY4ZjM)
```

File: Document-zhcn/0x02-Frontispiece.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+| --- | --- |
+| 中文 | Peter Chi, and Lex Chien, Henry Hu, Leo Wang |
+| 法语 | Romuald Szkudlarek, Christian Dong (审核) |
+| 德文 | Rocco Gränitz, Sven Schleier (审核) |
+| 西班牙语 | Martin Marsicano, Carlos Holguera |
+| 日语 | Koki Takeyama, Riotaro Okada (审核) |
+| 俄语 | Gall Maxim, Chelnokov Vladislav (审核), Oprya Egor (审核), Tereshin Dmitry (审核) |
+| 韩语 | Youngjae Jeon, Jeongwon Cho, Jiyou Han, Jiyeon Sung |
+
+本文档是Jim Manico编写的OWASP Application Security Verification Standard的分支
```
Comment (SpZanG) [inline Document-zhcn/0x02-Frontispiece.md]:
```suggestion
本文档是由Jim Manico编写的OWASP Application Security Verification Standard的一个分支。
```

File: Document-zhcn/0x02-Frontispiece.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+
+## 赞助商
+
+虽然`MASVS`和`MSTG`都是由社区自愿创建和维护的，但有时需要一点外部帮助。因此，我们感谢赞助商提供了能够聘请技术编辑的资金。请注意，他们的赞助不会以任何方式影响`MASVS`或`MSTG`的内容。赞助方案的描叙[OWASP项目Wiki](https://www.owasp.org/index.php/OWASP_Mobile_Security_Testing_Guide#tab=Sponsorship_Packages "OWASP Mobile Security Testing Guide Sponsorship Packages").
+
+### 尊敬的恩人
```
Comment (SpZanG) [inline Document-zhcn/0x02-Frontispiece.md]:
```suggestion
### 特别致谢
```

File: Document-zhcn/0x02-Frontispiece.md [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+| 西班牙语 | Martin Marsicano, Carlos Holguera |
+| 日语 | Koki Takeyama, Riotaro Okada (审核) |
+| 俄语 | Gall Maxim, Chelnokov Vladislav (审核), Oprya Egor (审核), Tereshin Dmitry (审核) |
+| 韩语 | Youngjae Jeon, Jeongwon Cho, Jiyou Han, Jiyeon Sung |
+
+本文档是Jim Manico编写的OWASP Application Security Verification Standard的分支
+
+## 赞助商
+
+虽然`MASVS`和`MSTG`都是由社区自愿创建和维护的，但有时需要一点外部帮助。因此，我们感谢赞助商提供了能够聘请技术编辑的资金。请注意，他们的赞助不会以任何方式影响`MASVS`或`MSTG`的内容。赞助方案的描叙[OWASP项目Wiki](https://www.owasp.org/index.php/OWASP_Mobile_Security_Testing_Guide#tab=Sponsorship_Packages "OWASP Mobile Security Testing Guide Sponsorship Packages").
```
Comment (SpZanG) [inline Document-zhcn/0x02-Frontispiece.md]:
```suggestion
虽然`MASVS`和`MSTG`都是由社区自愿创建和维护的，但是有时候仍然需要一点外部的帮助。因此，我们感谢我们的赞助商提供资金来聘请技术编辑。请注意，他们的赞助不会以任何方式影响`MASVS`或`MSTG`的内容。赞助方案的具体细节[OWASP项目Wiki](https://www.owasp.org/index.php/OWASP_Mobile_Security_Testing_Guide#tab=Sponsorship_Packages "OWASP Mobile Security Testing Guide Sponsorship Packages").
```

File: Document-zhcn/0x02-Frontispiece.md [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+
+### 尊敬的恩人
+
+[![NowSecure](images/NowSecure_logo.png)](https://www.nowsecure.com/ "NowSecure")
+
+接下来，我们要感谢OWASP海湾地区分会的赞助。最后，我们要感谢从Leanpub购买了这本书并以这种方式赞助我们的每个人。
```
Comment (SpZanG) [inline Document-zhcn/0x02-Frontispiece.md]:
```suggestion
接下来，我们要感谢OWASP海湾地区分会的赞助。最后，我们要感谢每一位从Leanpub购买了这本书并以这种方式赞助我们的人。
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 14]
```diff
@@ -0,0 +1,86 @@
+# 移动应用安全验证标准
+
+MASVS 可用于建立移动应用程序安全性的可信程度。制定需求时考虑到了以下目的：
```
Comment (SpZanG) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
MASVS 可以被用于确立移动应用程序的安全程度。此标准(MASVS)的建立遵循了下面三个主要目标：
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 15]
```diff
@@ -0,0 +1,86 @@
...
+
+- 可用于度量标准 - 提供一个安全标准，让开发人员和应用程序拥有者比较现有的移动应用程序;
```
Comment (SpZanG) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
- 可用于业界应用的安全度量标准 - 提供一个安全标准，让开发者和使用者们可以比较现有的移动应用程序。

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 16]
```diff
@@ -0,0 +1,86 @@
...
+- 可用作指导 - 在移动应用开发和测试的每个阶段提供指导;
```
Comment (SpZanG) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
- 可用于指导纲领 - 为移动应用程序整个开发阶段和测试提供安全指引；
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 17]
```diff
@@ -0,0 +1,86 @@
...
+- 可在采购过程中使用 - 为移动应用安全性验证提供基线。
```
Comment (SpZanG) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
可用于采购检验的验证参考 - 提供一个移动应用程序安全的验证标准。
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 18]
```diff
@@ -0,0 +1,86 @@
...
+
+## 移动应用安全模型 (Mobile AppSec Model)
+
+`MASVS`定义了两个安全验证级别（MASVS-L1 和 MASVS-L2），以及一组逆向工程弹性要求（MASVS-R）。 MASVS-L1 包含所有移动应用程序都建议使用的通用安全要求，而 MASVS-L2 应该应用于处理高度敏感数据的应用程序。如果设计目标是防止客户端威胁，则可以应用 MASVS-R覆 盖其他保护性控制。
```
Comment (SpZanG) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
`MASVS`定义了两个安全验证级别（MASVS-L1 和 MASVS-L2），以及一系列的抵抗逆向工程防御韧性准则（MASVS-R）。 MASVS-L1 是建议所有的移动应用程序都应该遵守的要求，MASVS-L2 则是针对处理高度敏感数据的移动应用程序。MASVS-R提供了需要防范客户端威胁时，可以使用的额外防护管控方法。
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 19]
```diff
@@ -0,0 +1,86 @@
...
+
+满足 MASVS-L1 中的要求将获得一个遵循安全最佳实践的安全应用程序，并且，不会遭受常见漏洞的困扰。 MASVS-L2 添加了其他深度防御控件（如SSL固定），从而使应用程序可以抵抗更复杂的攻击 - 假设移动操作系统的安全控件完好无损，并且最终用户不被视为潜在的威胁。满足MASVS-R中全部或部分软件保护要求，有助于阻止特定客户端威胁，包括用户恶意终端和/或移动操作系统被破坏。
```
Comment (SpZanG) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
满足 MASVS-L1 中所提到的要求，可以确保移动应用程序的安全，使开发流程符合信息安全的最佳实践方法，并且确保移动应用程序可以防御常见的安全问题。 MASVS-L2添加了额外的深度防御（defense-in-depth），例如凭证绑定（SSL pinning）。因此，MASVS-L2可以使移动应用程序能够更好的防范复杂的攻击行为 - 假设移动操作系统的安全控件完好无损，并且最终用户不被视为潜在的恶意用户。满足全部或者部分的MASVS-R软件保护要求，可以帮助阻止特定的客户端威胁，包括最终用户是恶意用户和手机操作系统遭到破坏。
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 20]
```diff
@@ -0,0 +1,86 @@
...
+
+**I：尽管我们建议每个移动应用程序中实施 `MASVS-L1` 管制，但是否实施安全治理最终应是基于风险的决策，该决策应与企业所有者沟通。**
```
Comment (SpZanG) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
**I：虽然我们建议应该在每一个移动应用程序中实施`MASVS-L1`准则，但是否实施最终应是基于风险的决策，该决策应与企业所有者沟通。**
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 21]
```diff
@@ -0,0 +1,86 @@
...
+
+**II：请注意，`MASVS-R` 中列出的软件保护管制以及<OWASP移动安全性测试指南>中所述的软件保护管制最终是可以被忽略的，绝不能用作安全管控的替代品。相反，它在为应用程序添加额外的，针对于威胁的保护控制，以满足MASVS-l1或MASVS- l2中的MASVS要求。**
```
Comment (SpZanG) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
**II：请注意，`MASVS-R`和<OWASP移动安全性测试指南>中所提到的软件保护准则无法确保不被绕过，绝对不能用作安全管控的替代品。相反的，这些软件保护准则旨在增加已落实MASVS-L1或MASVS- L2的移动应用程序，针对特定的威胁提供额外的防御。**
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 22]
```diff
@@ -0,0 +1,86 @@
...
+
+![验证级别](images/masvs-levels-new.jpg)
+
+### 文档结构
+
+MASVS的第一部分包含对安全模型和可用验证级别的描述，然后是有关如何在实践中使用该标准的建议。而在第二部分中，列出了详细的安全要求以及对对应的验证级别。根据技术目标/范围，将需求分为8类（V1至V8）。在整个MASVS和MSTG中使用以下术语：
```
Comment (SpZanG) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
MASVS的第一部分阐述了移动应用的安全模型和可用的安全验证级别，接着是有关如何使用本标准的建议。第二部分则是阐述了安全准则的细节和和对应的安全验证登记。基于技术的的目的/范畴，此准则被分成8个类别（从V1到V8）。准则类别与条目的命名方法如下（MASVS和MSTG使用相同的方法命名）：
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 23]
```diff
@@ -0,0 +1,86 @@
...
+
+-*要求类别:* MASVS-Vx，例如 MASVS-V2: 数据存储和隐私
```
Comment (SpZanG) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
-*准则类别：* MASVS-Vx，例如：MASVS-V2: 数据存储和隐私
```

File: Document-zhcn/0x03-Using_the_MASVS.md [hunk 24]
```diff
@@ -0,0 +1,86 @@
...
+-*要求:* MASVS-Vx.y，例如 MASVS-V2.2: "没有敏感数据写入应用程序日志"
```
Comment (SpZanG) [inline Document-zhcn/0x03-Using_the_MASVS.md]:
```suggestion
-*准则条目：* MASVS-Vx.y，例如 MASVS-V2.2: "敏感数据不应该被写入应用程序日志"
```

**Pull request closed without merging**